shared/teams/team/member-row/index.js

```diff
@@ -35,48 +47,61 @@ export const TeamMemberRow = (props: Props) => {
         padding: globalMargins.tiny,
         width: '100%',
       }}
-      onClick={active ? onClick : undefined}
     >
-      <Avatar username={username} size={isMobile ? 48 : 32} />
-      <Box style={{...globalStyles.flexBoxColumn, marginLeft: globalMargins.small}}>
-        <Box style={globalStyles.flexBoxRow}>
-          <Usernames
-            type="BodySemibold"
-            colorFollowing={true}
-            users={[{username, following, you: you === username}]}
-          />
-          {!active && (
-            <Meta
-              title="LOCKED OUT"
-              style={{background: globalColors.red, marginLeft: globalMargins.xtiny, marginTop: 4}}
+      <ClickableBox
+        style={{...globalStyles.flexBoxRow, flexGrow: 1, alignItems: 'center'}}
+        onClick={active ? onClick : undefined}
+      >
+        <Avatar username={username} size={isMobile ? 48 : 32} />
+        <Box style={{...globalStyles.flexBoxColumn, marginLeft: globalMargins.small}}>
+          <Box style={globalStyles.flexBoxRow}>
+            <Usernames
+              type="BodySemibold"
+              colorFollowing={true}
+              users={[{username, following, you: you === username}]}
             />
-          )}
-        </Box>
-        <Box style={globalStyles.flexBoxRow}>
-          {!!fullName &&
-            active && (
-              <Text style={{marginRight: globalMargins.xtiny}} type="BodySmall">
-                {fullName} •
-              </Text>
-            )}
-          {type &&
-            !!showCrown[type] && (
-              <Icon
-                // $FlowIssue "some string with unknown value"
-                type={'iconfont-crown-' + type}
-                style={{
-                  color: roleIconColorMap[type],
-                  fontSize: isMobile ? 16 : 12,
-                  marginRight: globalMargins.xtiny,
-                }}
+            {!active && (
+              <Meta
+                title="LOCKED OUT"
+                style={{background: globalColors.red, marginLeft: globalMargins.xtiny, marginTop: 4}}
               />
             )}
-          <Text type="BodySmall">
-            {active && type && typeToLabel[type]}{' '}
-            {!active && 'Has reset their account; admin(s) can re-invite'}
-          </Text>
+          </Box>
+          <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+            {!!fullName &&
+              active && (
+                <Text style={{marginRight: globalMargins.xtiny}} type="BodySmall">
+                  {fullName} •
+                </Text>
+              )}
+            {!!active &&
```

Comment:
lets not have complicated logic like this in jsx. instead hoist this up into its own var
```
const crown = active && type &&...
```
then just
```
{crown}
```

Reply:
i think if its a simple single check its fine but if its bigger than that likely too noisy